websites/C/Crowdin/dist/metadata.json

```diff
@@ -3,6 +3,12 @@
     "name": "TheDrop.",
     "id": "293828021134295040"
   },
+  "contributors": [
+    {
+      "name": "Rodry",
+      "id": "240875059953139714"
+    }
+  ],
```

Comment:
Changes not worth adding you as contributor.

Reply:
I literally changed the entire presence wdym lol

Reply:
Not really

Reply:
but ok